src/ChannelServer/Util/GmCommands.cs

```diff
@@ -49,6 +49,7 @@ public GmCommands()
 			Add("go", "<destination>", this.HandleGo);
 			Add("goto", "<team name>", this.HandleGoTo);
 			Add("recall", "<team name>", this.HandleRecall);
+			Add("recallall", string.Empty, this.HandleRecallAll);
```

Comment:
When all other commands use an empty string (`""`), a `string.Empty` sticks out a little bit, going against the convention. Btw, there's not much to actually gain from using `string.Empty`, it's just 10 more characters for no reason^^